dev_docs/tutorials/debugging.mdx

```diff
@@ -61,3 +61,49 @@ logging:
     - name: elasticsearch.query
       level: debug
 ```
+
+## Debugging Kibana with APM
+
+Kibana is integrated with APM's node and RUM agents. 
+To learn more about how APM works and what it reports, refer to the [documentation](https://www.elastic.co/observability/solution-overview).
+
+We currently track the following transactions from Kibana: 
+
+Backend (APM Node): 
+ * `request` - APM automatically tracks all incoming API requests
+ * `kibana-platform` - tracks server initiation phases take (preboot, setup and start)
+ * `task-manager` - tracks the operation of the task manager, including claiming pending tasks and marking them as running
+ * `task-run` - tracks the execution of individual tasks
+
+Frontend (APM RUM): 
```

Comment:
Is the frontend intentionally placed *after* the backend in the docs? Maybe it would be easier to read if we start with the frontend as requests originate there?